www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -72,192 +60,124 @@ There are two ways to download Xcode:
+cordova-ios includes a number of scripts that allow the platform to be used
+without the full Cordova CLI. This development path may offer you a greater
+range of development options in certain situations than the cross-platform CLI
+tool described in [The Command-Line Interface](../../cli/index.html).
+For example, you need to use shell tools when deploying a custom
+Cordova WebView alongside native components. Before using this
+development path, you must still configure the SDK environment
+as described in [Requirements and Support](#link-requirements-and-support)
+above.
 
-For further information, consult Apple's documentation:
+For each of the scripts discussed below, refer to
+[The Command-Line Interface](../../cli/index.html) for more information on their
+arguments and usage. Each script has a name that matches the corresponding CLI
+command. For example, `cordova-ios/bin/create` is equivalent to
+`cordova create`.
 
-*  [Start Developing iOS Apps Today](http://developer.apple.com/library/ios/#referencelibrary/GettingStarted/RoadMapiOS/index.html#//apple_ref/doc/uid/TP40011343) provides a quick overview of steps for developing iOS Apps.
+To get started, either download the cordova-ios package from
+[npm](https://www.npmjs.com/package/cordova-ios) or
+[Github](https://github.com/apache/cordova-ios).
 
-* [Member Center home page](https://developer.apple.com/membercenter/index.action)
-   provides links to several iOS technical resources including
-   technical resources, the provisioning portal, distribution guides
-   and community forums.
+To create a project using this package, run the `create` script in the `bin`
+folder:
 
-* [Tools Workflow Guide for iOS](http://developer.apple.com/library/ios/#documentation/Xcode/Conceptual/ios_development_workflow/00-About_the_iOS_Application_Development_Workflow/introduction.html#//apple_ref/doc/uid/TP40007959)
+    $ cordova-ios/bin/create ...
 
-* [Xcode User Guide](http://developer.apple.com/library/ios/#documentation/ToolsLanguages/Conceptual/Xcode4UserGuide/000-About_Xcode/about.html#//apple_ref/doc/uid/TP40010215)
+The created project will have a folder named `cordova` inside that contains
+scripts for the project-specific Cordova commands (e.g. `run`, `build`, etc.).
+Additionally, The project will feature a structure different from that of a
+normal Cordova project. Notably, `/www` is moved to `/assets/www`.
 
-* [Session Videos](https://developer.apple.com/videos/wwdc/2012/) from
-  the Apple World Wide Developer Conference 2012 (WWDC2012)
+To install plugins in this project, use the [Cordova Plugman Utility](../../../plugin_ref/plugman.html).
 
-* The [xcode-select command](http://developer.apple.com/library/mac/#documentation/Darwin/Reference/ManPages/man1/xcode-select.1.html),
-  which helps specify the correct version of Xcode if more than one is installed.
 
 (Mac®, OS X®, Apple®, Xcode®, App Store℠, iPad®, iPhone®, iPod® and  Finder® are Trademarks of Apple Inc.)
```

Comment:
As discussed, we should add an 'Upgrading' section here with reference to the upgrade article. The upgrade article shouldn't be deleted.